projects/plugins/jetpack/changelog/add-index-native-version-core-embed

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: other
+
+Embed block: Add native version of index file to control the available variations on the native version.
```

Comment:
Nitpick: I think because we already said we are adding the native version of the index file, we don't require the "on native version" at the end because that is a bit redundant. WDYT?

Reply:
I've applied this suggestion in [this commit](https://github.com/Automattic/jetpack/pull/21167/commits/322e4aad2c19b114c22eb39c7b4ecd0b84637489).